package.json

```diff
@@ -140,6 +147,7 @@
     "stylelint-config-standard": "18.2.0",
     "through2": "2.0.3",
     "vue-hot-reload-api": "2.1.0",
+    "vue-template-compiler": "2.4.4",
```

Comment:
Why has this been added?

Reply:
It was a wrong install. I was suppose to update `vueify`, that has `vue-template-compiler` as dep, to match with `vue` version. It's fixed now.